project/visualization/tests/test_utils.py

```diff
@@ -1,16 +1,18 @@
 from __future__ import unicode_literals
+
+import os
 from io import BytesIO
+
 import mock
 from PIL import Image as PILImage
 from PIL.Image import SAVE as PIL_SAVE
-
+from django.conf import settings
```

Comment:
These import changes are due to using the pycharm auto-format imports.

Reply:
Works for me.